bindings/matrix-sdk-ffi/src/timeline/mod.rs

```diff
@@ -201,14 +228,27 @@ pub struct UploadParameters {
     caption: Option<String>,
     /// Optional HTML-formatted caption, for clients that support it.
     formatted_caption: Option<FormattedBody>,
-    // Optional intentional mentions to be sent with the media.
+    /// Optional intentional mentions to be sent with the media.
     mentions: Option<Mentions>,
+    /// Optional parameters for sending the media as (threaded) reply.
+    reply_params: Option<ReplyParameters>,
```

Comment:
I've added this here to not have to duplicate the `send_*` methods for the various media types. Maybe we could also use this to merge `send_reply` and `send_thread_reply`?

Reply:
Yep, that would make sense (maybe as a followup PR, to not have this one drag for too long?)

Reply:
Separate PR makes sense. 👍